src/main/java/com/company/logic/EmployeeLogic.java

```diff
@@ -10,15 +13,31 @@
  */
 public class EmployeeLogic {
 
-    private static final String GET_BY_ID = "Employee.getById";
-
     public Employee get(int idEmployee) {
-        Employee employee = null;
+        Employee employee = new Employee();
+        SqlSession session = null;
+        try {
+            session = new Connection().getSqlSession();
+            if (session != null) {
+                employee = session.selectOne(ConstantData.GET_BY_ID_EMPLOYEE, idEmployee);
+                session.commit();
+            }
+        } catch (Exception e) {
```

Comment:
What kind of exceptions will be caught?
and... it would be ok not to return any error or message when an exception happens?